packages/shared/src/errors/RahaApiError/members/flag/MemberCantFlag.ts

```diff
@@ -2,37 +2,33 @@ import * as httpStatus from "http-status";
 
 import { RahaApiError } from "../..";
 
-export const ERROR_CODE = "flagMember.memberVerificationLevelTooLow";
-export interface MemberVerificationLevelTooLowErrorBody {
+export const ERROR_CODE = "flagMember.memberCantFlag";
+export interface MemberCantFlagErroBody {
```

Comment:
This can get repetitive if we have it for literally every possible action, can it be just a general "Can't do this" error with a variable "thing they can't do"?